### PR TITLE
Add image tag as optional input in ci task

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
This is complementary PR for the eq-pipelines changes in [this daily performance test PR](https://github.com/ONSdigital/eq-pipelines/pull/270).

### How to review 
Fly pipeline using these changes, check [this pipeline where I use image tag from the file](https://concourse.gcp.onsdigital.uk/teams/eq/pipelines/pete-daily-performance-test/jobs/run-benchmark/builds/24).

### CODE REVIEW

**Comment by petechd** [comment]
[Introduce EQ daily performance test (5 SP)](https://trello.com/c/wBIuJkpL/4869-introduce-eq-daily-performance-test-5-sp)

File: ci/README.md [hunk 1]
```diff
@@ -18,14 +18,15 @@ The following env vars must be set:
...
 - DOCKER_REGISTRY
-- IMAGE_TAG
 - USER_WAIT_TIME_MIN_SECONDS
 - USER_WAIT_TIME_MAX_SECONDS
 - PARALLELISM
 - TIMEOUT
 - OUTPUT_DIR
 - OUTPUT_BUCKET
 
+IMAGE_TAG can be passed as an environment variable or as a file input located at `./image-tag/tag`.
```
Comment (MebinAbraham) [inline ci/README.md:28]: The path needs updating.

**Review by MebinAbraham — APPROVED** [review]
As expected 👍

File: ci/README.md (hunk 1 above)
Comment (petechd) [reply ci/README.md]: Updated, thanks!

**Review by pricem14pc — APPROVED** [review]

**Pull request merged**